docs/.vuepress/config.js

```diff
@@ -119,9 +119,19 @@ module.exports = {
     lastUpdated: 'Last Updated',
   },
   dest: '_site',
-  plugins: ['@vuepress/google-analytics'],
+  plugins: [
+    '@vuepress/google-analytics',
+    [
+      '@vuepress/pwa',
+      {
+        serviceWorker: true,
+        updatePopup: true,
```

Comment:
remove updatePopup, and add `generateSWConfig`, which are the generateSW options from workbox